packages/augur-ui/src/modules/common/buttons.tsx

```diff
@@ -552,6 +558,30 @@ export const FilterButton = (props: DefaultActionButtonProps) => (
   </button>
 );
 
+export interface BettingBackLayButtonProps {
+  title?: string;
+  text?: string;
+  subText?: string;
+  action: Function;
+  disabled?: boolean;
+  type: BETTING_LAY | BETTING_BACK;
+}
+
+export const BettingBackLayButton = (props: BettingBackLayButtonProps) => (
```

Comment:
deconstruct props

Reply:
What is deconstruct props mean? Could you give me an example in the codebase?

Reply:
I think I got it.

Reply:
Updated. Thank you.